idea-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -49,4 +49,30 @@
             <action class="org.jetbrains.compose.desktop.ide.preview.RefreshOrRunPreviewAction"/>
         </group>
     </actions>
+
+
+    <extensions defaultExtensionNs="com.intellij">
+
+        <defaultLiveTemplates file="templates/ComposeMultiplatformTemplates.xml"/>
+
+        <intentionAction id="wrap_group" order="first">
+            <className>org.jetbrains.compose.intentions.WrapWithComposableIntentionGroup
+            </className>
+            <category>Composable intentions</category>
+        </intentionAction>
+
+        <intentionAction id="remove_composable" order="last">
+            <className>org.jetbrains.compose.intentions.RemoveComposableIntention
+            </className>
+            <category>Composable intentions</category>
+        </intentionAction>
+
+        <intentionAction id="remove_parent_composable" order="last">
+            <className>org.jetbrains.compose.intentions.RemoveParentComposableIntention
```

Comment:
Oh. I'm sorry, but I think you misunderstood the feature. 

Remove Composable removes everything - Selected and its children.
Remove Parent Composable just removes the Selected, but not its children.

In the above example, it'll only be available in case of Column, Row, and not on Box (since it doesnt have any children)